fixtures/bench_accumulator_factory.cpp.sem.uast

```diff
@@ -0,0 +1,3050 @@
+                                             offset: 136,
+                                             line: 11,
+                                             col: 19,
+                                          },
+                                       },
+                                       Name: "init",
+                                    },
+                                    Receiver: false,
+                                    Type: { '@type': "cpp:CPPASTSimpleDeclSpecifier",
+                                       '@token': "float",
+                                       '@role': [Number, Type],
+                                       '@pos': { '@type': "uast:Positions",
+                                          start: { '@type': "uast:Position",
+                                             offset: 126,
+                                             line: 11,
+                                             col: 9,
+                                          },
+                                          end: { '@type': "uast:Position",
+                                             offset: 131,
+                                             line: 11,
+                                             col: 14,
+                                          },
+                                       },
+                                       IsComplex: false,
+                                       IsConst: false,
+                                       IsConstExpr: false,
+                                       IsExplicit: false,
+                                       IsFriend: false,
+                                       IsImaginary: false,
+                                       IsInline: false,
+                                       IsLong: false,
+                                       IsLongLong: false,
+                                       IsRestrict: false,
+                                       IsShort: false,
+                                       IsSigned: false,
+                                       IsThreadLocal: false,
+                                       IsUnsigned: false,
+                                       IsVirtual: false,
+                                       IsVolatile: false,
+                                       StorageClass: "unspecified",
+                                    },
+                                    Variadic: false,
+                                 },
+                              ],
+                              Returns: [
+                                 { '@type': "uast:Argument",
+                                    Init: ~,
+                                    MapVariadic: false,
+                                    Name: ~,
+                                    Receiver: false,
+                                    Type: { '@type': "uast:Identifier",
+                                       Name: "unespecified",
```

Comment:
Looks like a void return. In this case, `Returns` should be `nil`.

Reply:
It's a constructor so it doesn't have a explicit `void` return (but implicitly it does, of couse), but yes, `Returns` should be `nil` in this case.

Reply:
Done